pkgs/applications/networking/msmtp/default.nix

```diff
@@ -10,11 +10,11 @@ let
 in stdenv.mkDerivation rec {
   pname = "msmtp";
   name = "${pname}-${version}";
```

Comment:
As I have just learned, you don't have to set this as long as pname and version are set.